cmd/gen_from_ddl.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.mercari.io/yo/generator"
+	"go.mercari.io/yo/internal"
+	"go.mercari.io/yo/loaders"
+)
+
+var (
+	genFromDDLOpts = internal.ArgType{}
+	genFromDDLCmd  = &cobra.Command{
+		Use:   "gen-from-ddl",
```

Comment:
The purpose of this command and the default command is same. I think it is better to be the same command.

How do you think to do like this?

* The command name is `generate` for both with using spanner and file.
* For the generation with spanner, it requires 3 arguments and it's the default behavior.
* With DDL, it requires a flag --from-ddl and an argument as the file path.

I'm not sure it's not confusing, but I want to consider to use the same command for both.

Reply:
OK.
I will rename subcommand name and add codes for using spanner instance.